backend/api/admin_views.py

```
@@ -0,0 +1,204 @@
+        
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)
+    #admin dlt functionality
+    def delete(self, request, video_id):
+        video = get_object_or_404(Video, id=video_id)
+        video.delete()
+        return Response({"message": "Video successfully deleted"}, status=status.HTTP_204_NO_CONTENT)
+
+class VideoStatsView(APIView):
+    permission_classes = [IsAuthenticated, IsAdmin]
+    
+    def get(self, request):
```

Ruff (0.8.2)

Missing return type annotation for public function `get`

(ANN201)